### PR TITLE
WIP  hepcrawl: new spider: PoS

### PR DESCRIPTION
**Pull request by ksachs**
* new spider for PoS harvesting, including sample_record and test
* modified split_fullname in utils.py
* modified affected spiders accordingly
* added language translation to utils and mappings

Signed-off-by: Kirsten Sachs <kirsten.sachs@desy.de>

### CODE REVIEW

**Comment by ksachs** [comment]
apart from missing fields in the data model (auxiliary_info) this is basically it.

File: hepcrawl/spiders/pos_spider.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+        date, year = self._get_date(node)
+        if date:
+            record.add_value('date_published', date)
+        if year:
+            record.add_value('journal_year', year)
+
+        identifier = node.xpath("//metadata//identifier/text()").extract_first()
+        if identifier:
+            pbn = re.split('[()]', identifier)
```
Comment (ksachs) [inline hepcrawl/spiders/pos_spider.py]: Is it OK to use re.split? Or do you prefer build-in functions?

File: hepcrawl/utils.py [hunk 2]
```diff
@@ -73,20 +74,26 @@ def collapse_initials(name):
...
         return "", ""
 
     if "," in author:
         fullname = [n.strip() for n in author.split(',')]
+        surname_first = True
     else:
         fullname = [n.strip() for n in author.split()]
+        surname_first = False
+
+#    surname_first = not switch_name_order == surname_first
```
Comment (ksachs) [inline hepcrawl/utils.py]: I prefer to spell this out (the following 2 lines), others prefer arithmetics.
Whatever you want.

File: hepcrawl/spiders/pos_spider.py (hunk 1 above)
Comment (jalavik) [inline hepcrawl/spiders/pos_spider.py]: This is OK 👍

File: hepcrawl/utils.py (hunk 2 above)
Comment (jalavik) [inline hepcrawl/utils.py]: I agree with your preference 👍

**Comment by jalavik** [comment]
Closing as agreed with @ksachs. Superseded by #21

**Pull request closed without merging**